src/app/pages/onboarding/welcome/welcome.layout.tsx

```diff
@@ -47,40 +48,65 @@ export function WelcomeLayout({
           </Box>
         </Flex>
         <Flex flexDir="column" alignItems={['normal', '', 'flex-start']}>
-          <LeatherButton
+          <Button
             invert={isAtleastBreakpointMd}
             mt={[0, 0, 'space.07']}
             onClick={onStartOnboarding}
             data-testid={OnboardingSelectors.SignUpBtn}
             aria-busy={isGeneratingWallet}
           >
             Create new wallet
-          </LeatherButton>
+          </Button>
           <Flex
             flexDir={['row', '', 'column']}
             gap="space.03"
             mt="space.04"
             alignItems="flex-start"
           >
-            <LeatherButton
-              variant={isAtleastBreakpointMd ? 'link' : 'outline'}
-              invert={isAtleastBreakpointMd}
-              flex={1}
-              mt={[0, 0, 'space.05']}
-              data-testid={OnboardingSelectors.SignInLink}
-              onClick={onRestoreWallet}
-            >
-              Use existing key
-            </LeatherButton>
-            <LeatherButton
-              variant={isAtleastBreakpointMd ? 'link' : 'outline'}
-              invert={isAtleastBreakpointMd}
-              flex={1}
-              mt={[0, 0, 'space.05']}
-              onClick={onSelectConnectLedger}
-            >
-              Use Ledger
-            </LeatherButton>
+            {isAtleastBreakpointMd ? (
```

Comment:
What do you think about stopping to use `useViewportMinWidth` hooks so much? 

When working on the new `Dialog` I noticed that sometimes it was causing re-render flashes when moving between views.

[Panda seem to recommend using `hideFrom` and `hideBelow` etc.](https://panda-css.com/docs/utilities/display) 

In this case we still need the hook to feed `invert` but I just wanted to raise the topic. 

So instead of:
```
{isAtleastBreakpointMd ? ( <X />) : (<Y />
```

We'd have:
```
<X hideBelow='md' />
<Y hideAbove='sm' />
```

Reply:
Thanks, sorry that was my off piste psuedo code. I should have checked it was working 😵‍💫